plugin/plugin.pro

```diff
@@ -18,4 +18,10 @@ HEADERS += \
         qrfilter.h \
         videofilterrunnable.h
 
-INSTALLS += target qmldir
+qmltypes.files += plugins.qmltypes
+qmltypes.path +=  $$target.path
+qmltypes.commands = qmlplugindump -nonrelocatable Amber.QrFilter 1.0 > $$PWD/plugins.qmltypes
+
+QMAKE_EXTRA_TARGETS += qmltypes
+
+INSTALLS += target qmldir qmltypes
```

Comment:
Ok to offer the plugin dump as extra target but let's skip it on default build. The dump can easily fail depending on the build environment.

Reply:
@alexb5 More importantly, the way qmlplugindump is invoked here would lead to a dump from the version installed under the build environment - not necessarily the same as the version being built. `qmlplugindump` needs to be run on a device instead, the result should be checked, possibly manually fixed (it is not 100% reliable) and then tracked under Git. `sfdk qmltypes` is there to help with that. See e.g. https://github.com/sailfishos/nemo-qml-plugin-notifications/blob/master/src/plugin/plugin.pro and `sfdk qmltypes --help` to learn how it makes use of the `qmltypes` make target.

Reply:
Ok. I got it! I will change PR soon.

Reply:
> @alexb5 More importantly, the way qmlplugindump is invoked here would lead to a dump from the version installed under the build environment - not necessarily the same as the version being built. `qmlplugindump` needs to be run on a device instead, the result should be checked, possibly manually fixed (it is not 100% reliable) and then tracked under Git. `sfdk qmltypes` is there to help with that. See e.g. https://github.com/sailfishos/nemo-qml-plugin-notifications/blob/master/src/plugin/plugin.pro and `sfdk qmltypes --help` to learn how it makes use of the `qmltypes` make target.

Shouldn't that be the same was on device if the version is installed in the build environment after?

Reply:
> Shouldn't that be the same was on device if the version is installed in the build environment after?
@martyone, could you please answer?

Reply:
@martyone Please take a look on PR. Is it possible to merge it?